mage_ai/data_preparation/models/block/utils.py

```diff
@@ -536,3 +538,34 @@ def fetch_input_variables(
                 input_vars[idx] = final_value
 
     return input_vars, kwargs_vars, upstream_block_uuids_final
+
+
+def get_spark_session():
```

Comment:
I would suggest creating a `spark` folder in https://github.com/mage-ai/mage-ai/tree/master/mage_ai/services
then put spark related methods in the file in that folder.

Reply:
> I would suggest creating a `spark` folder in https://github.com/mage-ai/mage-ai/tree/master/mage_ai/services then put spark related methods in the file in that folder.

Sure, made the suggested change.